### PR TITLE
Update to A-Frame v0.5.0

### PR DESCRIPTION
**Pull request by wmurphyrd**
No anticipated issues based on the A-Frame changelog. Tests pass and example works without any errors or warnings (Nightly and Chromium). 

Would you mind doing a release with the updated version? My packages that depend on this won't build with A-Frame v0.5.0 because of the peer dependency conflict. 

Thanks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "three-to-cannon": "^1.0.4"
   },
   "peerDependencies": {
-    "aframe": "^0.4.0"
+    "aframe": "^0.5.0"
   },
   "devDependencies": {
     "aframe": "^0.4.0",
```
Comment (donmccurdy) [inline package.json]: could you update this line as well?

**Comment by donmccurdy** [comment]
Looks safe to upgrade, yes. I should be able to cut a new release this weekend.

File: package.json (hunk 1 above)
Comment (wmurphyrd) [inline package.json]: Good catch

**Comment by wmurphyrd** [comment]
I saw #23, so, just to be sure, I ran the dist builds on this branch and rechecked the example with them. I'm not seeing any errors or issues with Chrome, FF, Chromium, or Nightly on Windows 10.

**Comment by donmccurdy** [comment]
Thanks!

**Pull request merged**